src/main/java/raisetech/StudentManagement/StudentCourse.java

```diff
@@ -0,0 +1,18 @@
+package raisetech.StudentManagement;
+
+import java.sql.Timestamp;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+
+public class StudentCourse {
+
+  private String id;
+  private String studentId;
+  private String courseName;
+  private Timestamp StartDate;
```

Comment:
変数名は小文字始まりで記述してください。
```suggestion
  private Timestamp startDate;
```